### PR TITLE
Loading wheels

### PR DESCRIPTION
**Pull request by hopeyen**

### CODE REVIEW

File: apps/frontend/src/components/ApproveRedirect.svelte [hunk 1]
```diff
@@ -5,6 +5,13 @@
   import SuccessIcon from './base/svg/SuccessIcon.svelte';
   import ErrorIcon from './base/svg/ErrorIcon.svelte';
   import Loader from './base/Loader.svelte';
+  // import { Circle } from 'svelte-loading-spinners';
```
Comment (Lev-Stambler) [inline apps/frontend/src/components/ApproveRedirect.svelte:8]: Lets remove commented out code. We generally don't want to merge commented code in

File: apps/frontend/src/components/ApproveRedirect.svelte [hunk 2]
```diff
@@ -5,6 +5,13 @@
...
+  import Spinner from 'svelte-spinner';
+  let size = 25;
+  let speed = 750;
+  let color = '#A82124';
```
Comment (Lev-Stambler) [inline apps/frontend/src/components/ApproveRedirect.svelte:12]: We don't really have anywhere to store color's right now but this is coming soon in a style PR! Could you just put a `//TODO: Change to global color vairable. See <issue link here>` above and open a Github issue for having common style/ color coding throughout all components?

File: apps/frontend/src/components/Auth.svelte [hunk 3]
```diff
@@ -15,6 +15,14 @@
   import { constants } from '../config/constants';
   import { skFromString } from '@baf-wallet/utils';
   import { reinitApp } from '../config/init.svelte';
+  import Spinner from 'svelte-spinner';
+  let size = 50;
+  let speed = 750;
+  let color = '#A82124';
```
Comment (Lev-Stambler) [inline apps/frontend/src/components/Auth.svelte:21]: Same thing for the color here

**Review by Lev-Stambler — CHANGES_REQUESTED** [review]
This looks great! Nice job getting going with the whole bunches of code. Just a few small things and we'll take our first steps to having a usable wallet!

**Review by hopeyen — COMMENTED** [review]
I think it has been changed! Github says changes are requested but I am not sure where else to see it, after yesterday's changes

**Comment by Sladuca** [comment]
> I think it has been changed! Github says changes are requested but I am not sure where else to see it, after yesterday's changes

You can re-request a review at the top by pressing the circular arrows button here
Screen Shot 2021-05-03 at 11 26 29 AM

File: package.json [hunk 4]
```diff
@@ -141,7 +141,9 @@
     "pm2": "^4.5.5",
     "reflect-metadata": "^0.1.13",
     "secp256k1": "^4.0.2",
+    "svelte-loading-spinners": "^0.1.3",
     "svelte-spa-router": "^3.1.0",
+    "svelte-spinner": "^2.0.1",
```
Comment (Lev-Stambler) [inline package.json:146]: Looks great!

**Review by Lev-Stambler — APPROVED** [review]
Looks great!

**Pull request merged**